runtime/arc.js

```diff
@@ -251,6 +251,11 @@ class Arc {
   }
 
   _registerView(view, tags) {
+    tags = tags || [];
+    tags = Array.isArray(tags) ? tags : [tags];
+    tags.forEach(tag => assert(tag.startsWith('#'),
```

Comment:
I think we don't actually know that tag is a String here. Is it too granular validation to check? Whatever similar code is doing in area is fine.

Reply:
Now I'm questioning my whole approach :)

It may be that we don't care if the tag is a string as long as it has the properties that we're looking for (can be a key, can be serialized, can be written into a manifest, etc). That feels like some leveraging of a dynamic type system (and for the same reason, Array.isArray() feels like too strict of a check but it's the closest I saw).

I wonder if a better approach would have been to instead find where tags without a # broke, and fixed that?

I'm going to ignore all of this, bury my head in the sand here, and merge.